test/powershell/engine/Help/HelpSystem.Tests.ps1

```diff
@@ -225,4 +225,29 @@ Describe "Get-Help should find help info within help files" -Tags @('CI', 'Requi
             Remove-Item $helpFilePath -Force -ErrorAction SilentlyContinue
         }
     }
+
+    It "Get-Help should find pattern help files" {
+        $helpFile = "about_testCase.help.txt"
+        $culture = (Get-Culture).Name
+        $helpFileWithPattern = "about*.help.txt"
+        $helpFolderPath = Join-Path $PSHOME $culture
+        $helpFilePath = Join-Path $helpFolderPath $helpFile
+
+        if (!(Test-Path $helpFolderPath))
+        {
+            $null = New-Item -ItemType Directory -Path $helpFolderPath -ErrorAction SilentlyContinue
+        }
+
+        try
+        {
+            # Create at least one help file matches "about*" pattern
+            $null = New-Item -ItemType File -Path $helpFilePath -Value "about_test" -ErrorAction SilentlyContinue
+            (Get-ChildItem $PSHOME\$helpFileWithPattern -Recurse).Count | Should Be (Get-Help about*).Count
```

Comment:
Get-Help can get help content from other paths under PSModulePath. So if there are other about help topics not under $PSHome, the counts would not match.